exercises/largest-series-product/src/example.h

```diff
@@ -0,0 +1,8 @@
+#ifndef _LARGEST_SERIES_PRODUCT_H
+#define _LARGEST_SERIES_PRODUCT_H
+
+#include <stddef.h>
+
+long largest_series_product(char *digits, size_t span);
```

Comment:
Isn't `long` usually 32bit? Looks like we need to use `int64_t` here for the return type.